test/e2e/edu-5490-helpers.js

```diff
@@ -0,0 +1,150 @@
+import _ from 'lodash/fp';
+const Timeouts = require('./timeouts.js');
+
+
+// Same as 1990e
+// TODO: relocate to keep it DRY
```

Comment:
Any reason not to move the identical function in `test/e2e/edu-1990e-helpers.js` into `test/e2e/edu-helpers.js` now instead of later?